stock_move_description/stock_move.py

```diff
@@ -19,44 +19,42 @@
 #
 ##############################################################################
 
-from openerp.osv import orm, fields
+from odoo import fields, models
 
 
-class StockMove(orm.Model):
+class StockMove(models.Model):
     _inherit = "stock.move"
+    name = fields.Text(string="Description", required=False, )
 
-    _columns = {
-        'name': fields.text('Description'),
-    }
 
     def onchange_product_id(
-            self, cr, uid, ids, prod_id=False, loc_id=False,
+            self, prod_id=False, loc_id=False,
             loc_dest_id=False, partner_id=False, context=None
     ):
         context = dict(context or {})
         res = super(StockMove, self).onchange_product_id(
-            cr, uid, ids, prod_id=prod_id, loc_id=loc_id,
+             prod_id=prod_id, loc_id=loc_id,
             loc_dest_id=loc_dest_id, partner_id=partner_id
         )
         if prod_id:
-            user = self.pool.get('res.users').browse(
-                cr, uid, uid, context=context)
+            user = self.env['res.users'].browse(
+                 context=context)
```

Comment:
@AbdulrhmanGad Context is not passed through function arguments. If you want current user, you can use self.env.user